modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+// persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "persistent-storage-csi-vsphere"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:azure:
+endif::[]
+
+[id="csi-tp-enable_{context}"]
+= Enabling the {FeatureName} CSI driver operator
+To enable the {FeatureName} Container Storage Interface (CSI) driver operator, you must enable feature gates.
+
+.Procedure
+
+. Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#[Enabling features using feature gates]).
```

Comment:
Same comment here - should we be explicit that it's the `TechPreviewNoUpgrade` feature gate? I see multiple listed in the referenced assembly.

Reply:
Yes!